atomicapp/cli/main.py

```diff
@@ -365,6 +375,21 @@ def create_parser(self):
             help='The name of a container image containing an Atomic App.')
         gena_subparser.set_defaults(func=cli_genanswers)
 
+        # === "init" SUBPARSER ===
+        init_subparser = toplevel_subparsers.add_parser(
+            "init", parents=[globals_parser])
```

Comment:
yeah leave it like it is for now. We have a lot to do around handling of both config and CLI :(.